core/esmf-aspect-meta-model-python/esmf_aspect_meta_model_python/loader/instantiator/state_instantiator.py

```diff
@@ -0,0 +1,103 @@
+#  Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+#
+#  See the AUTHORS file(s) distributed with this work for additional
+#  information regarding authorship.
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+#   SPDX-License-Identifier: MPL-2.0
+
+import typing
+
+import rdflib  # type: ignore
+
+from rdflib.term import Node
+
+from esmf_aspect_meta_model_python.base.characteristics.state import State
+from esmf_aspect_meta_model_python.impl.characteristics.default_state import DefaultState
+from esmf_aspect_meta_model_python.loader.instantiator.constants import DATA_TYPE_ERROR_MSG
+from esmf_aspect_meta_model_python.loader.instantiator_base import InstantiatorBase
+from esmf_aspect_meta_model_python.loader.rdf_helper import RdfHelper
+from esmf_aspect_meta_model_python.vocabulary.SAMM import SAMM
+from esmf_aspect_meta_model_python.vocabulary.SAMMC import SAMMC
+
+
+class StateInstantiator(InstantiatorBase[State]):
+    def _create_instance(self, element_node: Node) -> State:
+        data_type = self._get_data_type(element_node)
+        if data_type is None:
+            raise TypeError(DATA_TYPE_ERROR_MSG)
+
+        meta_model_base_attributes = self._get_base_attributes(element_node)
+        value_collection_node = self._aspect_graph.value(
+            subject=element_node,
+            predicate=self._sammc.get_urn(SAMMC.values),
+        )
+        value_nodes = RdfHelper.get_rdf_list_values(value_collection_node, self._aspect_graph)
+        values = [self.__to_state_node_value(value_node) for value_node in value_nodes]
+
+        defaultValue = self._aspect_graph.value(
+            subject=element_node,
+            predicate=self._sammc.get_urn(SAMMC.default_value),
+        )
+        default = self.__to_state_node_value(defaultValue)
+
+        return DefaultState(meta_model_base_attributes, data_type, values, default)
+
+    def __to_state_node_value(self, value_node: Node) -> typing.Dict:
```

Comment:
value_node can be None
so, definition can update with 
`def __to_state_node_value(self, value_node: Optional[Node])`

in the method **_create_instance**
'
        defaultValue = self._aspect_graph.value(
            subject=element_node,
            predicate=self._sammc.get_urn(SAMMC.default_value),
        )
        default = self.__to_state_node_value(defaultValue)
'
self._aspect_graph.value can return None for the defaultValue